programs/fuzz/common.c

```diff
@@ -4,11 +4,13 @@
 #include <stdlib.h>
 #include "mbedtls/ctr_drbg.h"
 
+#if defined(MBEDTLS_HAVE_TIME)
```

Comment:
If you look at line 18, `dummy_constant_time` is only used when `MBEDTLS_PLATFORM_TIME_ALT` is defined, so this should be `#if defined(MBEDTLS_PLATFORM_TIME_ALT)`.

There is a similar issue in `ssl_test_lib.[ch]`.

(TBH, since `dummy_constant_time()` is only used by `dummy_init()`, I would have it `static` and not in the `.h` file, but that's a bigger set of changes, and I wouldn't want to start pulling on that thread)